de.metas.business.rest-api/src/main/java/de/metas/rest_api/ordercandidates/request/JsonOLCandCreateRequest.java

```diff
@@ -51,29 +53,30 @@
 @Value
 public final class JsonOLCandCreateRequest
 {
-	JsonOrganization org;
+	private JsonOrganization org;
 
 	@ApiModelProperty( //
-			required = true, //
-			value = "This translates to `C_OLCand.externalLineId`.\n"
-					+ "`externalLineId` and `dataSourceInternalName` together need to be unique.")
+
+			allowEmptyValue = false, //
```

Comment:
please check the javadoc for  `allowEmptyValue`; afair it's anyways `false`be default, and `required = true` was correct;

Reply:
fixed